### PR TITLE
OBSDOCS-603: Add logging attributes

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-603

Link to docs preview:
Fixed attribute module: https://67730--docspreview.netlify.app/openshift-enterprise/latest/logging/log_visualization/log-visualization#log-visualization-resource-logs

QE review:
N/A

Additional information:

Adding attributes for logging. There will be follow up PRs to replace these across the docs.
Also taking this opportunity to fix an attribute that's broken in one of the modules.

### CODE REVIEW

File: modules/viewing-resource-logs-cli-console.adoc [hunk 1]
```diff
@@ -7,11 +7,11 @@
 [id="viewing-resource-logs-cli-console_{context}"]
 = Viewing resource logs
 
-You can view the log for various resources in the {first-oc} and web console. Logs read from the tail, or end, of the log.
+You can view the log for various resources in the {oc-first} and web console. Logs read from the tail, or end, of the log.
 
 .Prerequisites
 
-* Access to the {first-oc}.
+* Access to the {oc-first}.
```
Comment (mjpytlak) [inline modules/viewing-resource-logs-cli-console.adoc:14]: Given that you already introduce `OpenShift CLI (oc)` with `{oc-first}` in line 10, I do not think you need `{oc-first}` at line 14. Do we have an attribute that can be used for subquent references to `oc`?

**Review by mjpytlak — COMMENTED** [review]
Quick observation @abrennan89

File: modules/viewing-resource-logs-cli-console.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/viewing-resource-logs-cli-console.adoc:14]: Pretty sure that's the only one

**Pull request merged**